packages/ui/src/components/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { ToggleGroup, ToogleGroupOption } from "./ToggleGroup";
+import { useState } from "react";
+import { fn } from "@storybook/test";
+
+const meta = {
+  title: "Swapr-ui/ToogleGroup",
+  component: ToggleGroup,
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+  argTypes: {
+    size: { control: "radio", options: ["xs", "sm", "md", "lg"] },
```

Comment:
We have an issue with this prop, because it's a prop applied to `ToogleGroupOption` and not to `ToggleGroup`

Reply:
How can we add it to `ToogleGroupOption`?

Reply:
AFAIK you can't, you should do a ToggleGroupOption story. There was a way to place [subcomponents on storybook ](https://storybook.js.org/docs/writing-stories/stories-for-multiple-components#subcomponents) but the time I tried to use it, it was working kinda funky